jbmc/src/java_bytecode/java_types.h

```diff
@@ -111,6 +111,16 @@ class java_class_typet:public class_typet
     return set(ID_access, access);
   }
 
+  const irep_idt &get_inner_class() const
```

Comment:
Return a bool (using `get_bool` as done in `get_final`) as this hides the messiness that everything is strings from the user of this function.